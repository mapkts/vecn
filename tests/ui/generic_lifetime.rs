#[vecn::vector]
struct TupleStruct<'a>(&'a f32);

#[vecn::vector]
struct NamedStruct<'a> {
    x: &'a f32,
    y: &'a f32,
}

fn main() {}
