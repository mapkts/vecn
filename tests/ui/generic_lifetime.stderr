error: expected generic type, but found generic lifetime
 --> $DIR/generic_lifetime.rs:2:20
  |
2 | struct TupleStruct<'a>(&'a f32);
  |                    ^^

error: expected generic type, but found generic lifetime
 --> $DIR/generic_lifetime.rs:5:20
  |
5 | struct NamedStruct<'a> {
  |                    ^^
